src/page-components/Home/index.tsx

```diff
@@ -49,17 +49,18 @@ export const Home = () => {
       </div>
       <PageSection className="Trk__flex Trk__direction--column Trk__justify--center Trk__align--center">
         <div className= "font__weight--xbold Trk__LineHeight--6 Trk__text--center">
-          <p className="Trk__mt--1 Trk__mb--1">
+          <p>
```

Comment:
In this home page component, extract the JSX that renders the social media icons into a separate component. Export that component for use in this page. Test that component to make sure it has all of the required social media components.